### DISCUSSION TITLE
FEATURE REQUEST: Customize feed length

### DISCUSSION THREAD

**Discussion by LuisFelipeQuesada** [dailydotdev/daily#969] [Feature Requests]
Instead of having infinite feed, to have the possibility of having pages would be nice. This can be an option, to have infinite feed or to have it paginated.

Thanks for the effort made on the app!!

**Comment by idoshamun** [comment]
Hi @LuisFelipeQuesada,

What would be the advantage or the use case for this in your opinion?
Thanks for the suggestion!
